waspc/.stylish-haskell.yaml

```diff
@@ -0,0 +1,230 @@
+# stylish-haskell configuration file
```

Comment:
I added this file so that stylish haskell can work for us, it is config file, same as we have for hlint.